app/locales/nl.js

```diff
@@ -34,82 +34,82 @@ export default {
   loading: '(Laden)',
   classifier: {
     back: 'Terug',
-    backButtonWarning: 'Als je terug gaat verlies je je werk voor de huidige taak.',
-    close: 'Close',
-    continue: 'Continue',
-    done: 'Klaar',
-    doneAndTalk: 'Klaar & Praat',
-    dontShowMinicourse: 'Do not show mini-course in the future',
-    letsGo: 'Let’s go!',
-    next: 'Verder',
-    optOut: 'Opt out',
+    backButtonWarning: 'Let op! Niet opgeslagen gegevens gaan verloren.',
+    close: 'Sluiten',
+    continue: 'Doorgaan',
+    done: 'Afronden',
+    doneAndTalk: 'Afronden & Bespreken',
+    dontShowMinicourse: 'Verberg de uitleg in het vervolg.',
+    letsGo: 'Laten we beginnen!',
+    next: 'Volgende',
+    optOut: 'Opt-out',
     taskTabs: {
-      taskTab: 'Task',
-      tutorialTab: 'Tutorial'
+      taskTab: 'Taak',
+      tutorialTab: 'Uitleg'
     },
-    recents: 'Je recente classificaties',
-    talk: 'Praat',
-    taskHelpButton: 'Need some help with this task?',
-    miniCourseButton: 'Herstart de minitour'
+    recents: 'Uw recente waarnemingen',
```

Comment:
Up for discussion: do we want to use the Dutch formal "you" (`u`) or the less-formal "you" (`je`)?

Reply:
@mrniaboc might be the best to decide this i guess?